app/controllers/general.php

```diff
@@ -316,21 +283,26 @@ function router(App $utopia, Database $dbForPlatform, callable $getProjectDB, Sw
             'errors' => '',
             'logs' => '',
             'duration' => 0.0,
-            'search' => implode(' ', [$resourceId, $executionId]),
+            'search' => implode(' ', [$resource->getId(), $executionId]),
         ]);
 
         if ($type === 'function') {
             $execution->setAttribute('resourceType', 'functions');
             $execution->setAttribute('trigger', 'http'); // http / schedule / event
             $execution->setAttribute('status', 'processing'); // waiting / processing / completed / failed
+
+            $queueForEvents
+                        ->setParam('functionId', $resource->getId())
+                        ->setParam('executionId', $execution->getId())
+                        ->setContext('function', $resource);
         } elseif ($type === 'site') {
             $execution->setAttribute('resourceType', 'sites');
-        }
 
-        $queueForEvents
-            ->setParam('functionId', $resource->getId())
-            ->setParam('executionId', $execution->getId())
-            ->setContext('function', $resource);
+            $queueForEvents
+                ->setParam('siteId', $resource->getId())
+                ->setParam('executionId', $execution->getId())
```

Comment:
Sites have `executionId`?

Reply:
Functions have the same piece of code. Yes, site's logs are in execution collection, exactly same as functions